src/shared/utils/utils.ts

```diff
@@ -10,11 +10,9 @@ import * as v3 from "../../__generated__/schema.3.0";
 import { WrappedDocument as WrappedDocumentV3 } from "../../3.0/types";
 import { OpenAttestationDocument as OpenAttestationDocumentV3 } from "../../__generated__/schema.3.0";
 
-import * as v4 from "../../__generated__/schema.4.0";
-import { WrappedDocument as WrappedDocumentV4 } from "../../4.0/types";
-import { OpenAttestationDocument as OpenAttestationDocumentV4 } from "../../__generated__/schema.4.0";
+import { V4WrappedDocument as WrappedDocumentV4 } from "../../4.0/types";
```

Comment:
Understand you changed the naming convention of v4 types to `V_WrappedDocument` in `4.0/types.ts`. Is there a reason to alias this import back to to `WrappedDocumentV4` here?